src/sagas/nanoContract.js

```diff
@@ -75,6 +75,9 @@ export function* init() {
 export function* registerNanoContract({ payload }) {
   const { address, ncId } = payload;
 
+  // This delay protects this process against user jitter
+  yield delay(720);
```

Comment:
I introduced it because it runs quickly and in case a Nano Contract is already registered one can retry retry many times quickly. But maybe it is something unnecessary. I'm removing it.